lib/nimble.phx.gen.template/template.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Nimble.Phx.Gen.Template.Template do
+  alias Nimble.Phx.Gen.Template.{Project, Addons}
+  alias Nimble.Phx.Gen.Template.Api.Template, as: ApiTemplate
+  alias Nimble.Phx.Gen.Template.Web.Template, as: WebTemplate
+
+  def apply(%Project{} = project) do
+    project = shared_setup(project)
+
+    mix_format()
```

Comment:
Why is it needed to be run first at the beginning? Would be good to document why a comment.

Reply:
We don't need that anymore @olivierobert